### DISCUSSION TITLE
How to add a homepage access password

### DISCUSSION THREAD

**Discussion by hyy34** [YOURLS/YOURLS#3641] [Q&A]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/YOURLS/.github/blob/main/CODE_OF_CONDUCT.md).

### Submission validity

- [X] This is not a personal support request, that should be posted on [discussions community](https://github.com/YOURLS/YOURLS/discussions).
- [X] I checked [current issues](https://github.com/YOURLS/YOURLS/issues) and this request isn't a duplicate of an existing issue, opened or closed.

### Summary

For personal use, you can add administrator page access account and password, how to add home page access password?

### Motivation

It is possible to make it impossible to set up short links for people who are not me

**Comment by dgw** [comment]
This is more or less the same question as #2029.

YOURLS doesn't come with a homepage, period. If you create an `index.php` for your installation, it's _your_ job to restrict access if necessary using IP restrictions, HTTP Basic Auth, etc.

You can find tutorials for setting those up with a web search for the name of your web server software (e.g. "nginx" or "apache") + the type of security you want (e.g. "http basic auth" or "ip restrictions").
